config.go

```diff
@@ -891,6 +896,16 @@ func loadConfig() (*config, []string, error) {
 		maxPerBlock = cfg.TicketBuyFreq
 	}
 
+	// Use --ticketmaxprice setting if it has been set and --ticketbuyer.maxpriceabsolute
+	// has not been changed.
+	maxPriceAbsolute := cfg.TBOpts.MaxPriceAbsolute
```

Comment:
This should be an Amount not an AmountFlag (which it is currently it shares the same type as MaxPriceAbsolute)